src/types.ts

```diff
@@ -1,102 +1,93 @@
-/* eslint-disable max-lines */
-
 // local
 import type { Remote } from 'comlink';
-import penumbra from './API';
-import { PenumbraError } from './error';
+import type { PenumbraWorker as PenumbraWorkerAPI } from './worker.penumbra';
+import type { PenumbraError } from './error';
+
+import type { JobID } from './job-id';
+import type { StreamSaverEndpoint } from './streamsaver';
 
 export { PenumbraZipWriter } from './zip';
 
 /**
  * penumbra.encrypt() encryption options config (buffers or base64-encoded strings)
  */
 export interface PenumbraEncryptionOptions {
-  /** Encryption key */
-  key: string | Buffer;
+  /** Encryption key, either Uint8Array or base64-encoded string */
+  key: Uint8Array | string;
+  /** Initialization vector, either Uint8Array or base64-encoded string */
+  iv: Uint8Array | string;
 }
```

Comment:
Not sure how this corresponds to real life usage, but if we can't mix strings and Uint8Arrays (e.g. an Uint8Array `key` and a string `iv`), would it make sense to use a union type instead?

```ts
type PenumbraEncryptionOptions =
  | { key: Uint8Array; iv: Uint8Array } 
  | { key: string; iv: string } 
```